packages/roosterjs-content-model-dom/lib/formatHandlers/utils/color.ts

```diff
@@ -59,14 +61,23 @@ export function getColor(
     darkColorHandler?: DarkColorHandler,
     fallback?: string
 ): string | undefined {
-    let color =
-        (isBackground ? element.style.backgroundColor : element.style.color) ||
-        element.getAttribute(isBackground ? 'bgcolor' : 'color') ||
-        fallback;
-
-    if (color && DeprecatedColors.indexOf(color) > -1) {
-        color = isBackground ? undefined : BlackColor;
-    } else if (darkColorHandler && color) {
+    const color = retrieveElementColor(element, isBackground ? 'background' : 'text', fallback);
+
+    return color ? getLightModeColor(color, isBackground, isDarkMode, darkColorHandler) : undefined;
+}
+
+/**
+ * @internal
+ */
+export function getLightModeColor(
+    color: string,
+    isBackground: boolean,
```

Comment:
I suggest we don't pass in `isBackground` to this function. This function should just be used for getting light color, it should not know any knowledge about if it is background or others. For `DeprecatedColors`, you can pass in a fallback color, for example:

function getLightModeColor(color, isDarkMode, darkModeHandler, fallback?: string) {
  if (is deprecated color) {
    return fallback;
  }
 ..
}

Then pass it in from outside:

return color ? getLightModelColor(color, isDarkMode, darkColorHandler, isBackground ? undefined : BackColor);

So this function can be used from any scenario